torch/_inductor/codegen/wrapper.py

```diff
@@ -1881,6 +1881,8 @@ def add_arg(idx, arg, is_constexpr=False, equals_1=False, equals_none=False):
                 "kwargs": config.kwargs,
                 "num_warps": config.num_warps,
                 "num_stages": config.num_stages,
+                "num_consumer_groups": getattr(config, 'num_consumer_groups', 0),
```

Comment:
Is this only for an internal version of triton ? I don't see these fields as parameters on triton's config: 

https://github.com/triton-lang/triton/blob/18b8d1d0f44cd641b0ac6f041bda5a7d747b9acc/python/triton/runtime/autotuner.py#L251-L269 ?

or do you know where this is documented in triton repro ?

Reply:
You are correct, this is only applicable to the facebookexperimental/ws version (https://github.com/facebookexperimental/triton/blob/ws/python/triton/runtime/autotuner.py#L268)